.github/workflows/build-and-test.yml

```diff
@@ -112,7 +113,7 @@ jobs:
       fail-fast: false
       matrix:
         node-version: [20, 22, 24]
-        os: [macos-14, ubuntu-latest, windows-latest, ubuntu-24.04-arm]
+        os: [macos-14, macos-15-intel, ubuntu-latest, windows-latest, ubuntu-24.04-arm]
```

Comment:
Is there a reason you went with the intel variant, rather than `macos-15` (arm64): https://github.com/actions/runner-images/tree/main?tab=readme-ov-file#available-images

Reply:
i changed it to macos-15 as on the pact-js pr josh had asked to replace macos-13 with macos-15-intel https://github.com/pact-foundation/pact-js/pull/1628

Reply:
Ah, I see. That project already had `macos-latest`. 

Let's add that too in this section alongside the intel variant, update the `prebuilds` section to use `macos-latest` (not intel) and then I think we're good to merge.